ISISComputingGroup/ibex_bluesky_core
# 21 generate dae uncertainties

Exposes signals for summed dae counts uncertainties.

To test;
- Check that all unit tests pass.
- Check that the `dae_test` manual test runs and that the uncertainty is the sqrt of normalised counts.
